examples/src/demos/Tear.tsx

```diff
@@ -0,0 +1,121 @@
+import { createContext, createRef, useContext } from 'react'
+import { Canvas } from '@react-three/fiber'
+import { Physics, useSphere, useBox, useConeTwistConstraint } from '@react-three/cannon'
+
+import type { PropsWithChildren } from 'react'
+import type { Triplet } from '@react-three/cannon'
+import type { Object3D } from 'three'
+
+const parent = createContext({
+  ref: createRef<Object3D>(),
+  pos: [0, 0, 0] as Triplet,
+})
+
+const ChainLink = ({ children, ...props }: PropsWithChildren<{}>) => {
```

Comment:
Most of this is copied & modified version of https://github.com/pmndrs/use-cannon/blob/master/examples/src/demos/Chain.tsx#L9-L11

With a bit of work they could reuse the same components, with `Chain.tsx` not leveraging all of the features to demonstrate tear.

Alternatively, `Chain.tsx` and `Tear.tsx` could become 1 example with some interactivity.

Let me know your preference.